apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -306,38 +479,45 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                 {
                     step !== 3
                         ? (
-                            < Modal.Header className="totp-header">
-                                <div className="illustration">
-                                    <GenericIcon
-                                        transparent
-                                        size="small"
-                                        icon={ getQRCodeScanIcon() }
-                                    />
-                                </div>
+                            < Modal.Header className="wizard-header text-center">
+                                Set Up An Authenticator App
                             </Modal.Header>
                         )
                         : null
                 }
                 <Modal.Content data-testid={ `${ testId }-modal-content` } scrolling>
                     { stepContent(step) }
                 </Modal.Content>
-                <Modal.Actions data-testid={ `${ testId }-modal-actions` }>
+                <Modal.Actions data-testid={ `${ testId }-modal-actions` } className ="actions">
                     {
-                        step !== 1
+                        step === 0
                             ? (
-                                < Button onClick={ () => { setOpenWizard(false); } } className="link-button"
-                                  data-testid={ `${ testId }-modal-actions-cancel-button`}>
-                                    { t("common:cancel") }
-                                </Button>
-                            )
-                            : null
-                    }
-                    <Button onClick={ () => { handleModalButtonClick(step); } } primary
-                        data-testid={ `${ testId }-modal-actions-primary-button`}>
-                        { stepButtonText(step) }
-                    </Button>
+                        <Message className="totp-tooltip display-flex">      
+                            <Icon name="info circle" />
+                            <Message.Content> 
+                            <Trans
+                                i18nKey={ (translateKey + "modals.toolTip") }
+                            >
+                                Don&apos;t have an app? Download an authenticator 
+                                application like Google Authenticator from 
+                                <a href="https://www.apple.com/us/search/totp?src=globalnav"> App Store </a>
```

Comment:
This will open up the link in the same tab. Is it the desired behavious. If you decide to add `target="_blank"` use `rel="noopener noreferrer"`.

https://clever-solution.com/blog/everything-you-need-to-know-about-rel-noopener-noreferrer-tags-purpose-benefits-and-seo-impact

```suggestion
                                <a href="https://www.apple.com/us/search/totp?src=globalnav"> App Store </a>
```